building/tracks/concept-exercises.md

```diff
@@ -322,18 +317,8 @@ Assume that the user FSharpForever has written an exercise called `basics` for t
 
 ```json
 {
-  "contributors": [
-    {
-      "github_username": "PythonPerfection",
-      "exercism_username": "PythonPerfection"
-    }
-  ],
-  "authors": [
-    {
-      "github_username": "PythonProfessor",
-      "exercism_username": "PythonProfessor"
-    }
-  ],
+  "contributors": ["PythonPerfection"],
+  "authors": ["PythonProfessor"],
```

Comment:
Could we change this second example so it's more visibly different to the first pls?

Reply:
Something like this (pushed change)?